numerics_paper/version2/numerical_chj.tex

```diff
@@ -570,13 +564,11 @@ \subsection{Blended Riemann solver}\label{sec:blended_rs}
 \end{subequations}
 
 
-\subsection{Entropy-stable Riemann solver}\label{sec:entropy_stable}
-The Riemann solver presented in the previous section blends the solvers of Rusanov and Roe based on a
-local measurement of the entropy residual. The objective is to induce dissipation of entropy
-near discontinuities. However, we have no guarantee that methods \eqref{first-order_via_fluct}
-or \eqref{second-order_via_fluct} based on this Riemann solver are (locally) entropy stable.
-In this section, we further modify the first-order method
-\eqref{first-order_via_fluct} in order to guarantee local entropy stability.
+\subsection{Entropy stabilization}\label{sec:entropy_stable}
+In the previous section we neglected the term $\lambda^{\min}_\iph$ in
+\eqref{blended-flux-1}.  In this section, we show how this term is computed,
+in order to guarantee local entropy stability of the scheme given by \eqref{blended-flux-1}
```

Comment:
Again, I think this reference is not correct.

Reply:
What should it refer to?

Reply:
I think (8), but I am not sure if you have something else in mind that I am not considering.